docs/workforce-management-framework/topics/con-PassportAuth-Module.adoc

```diff
@@ -0,0 +1,38 @@
+[]
+= {WFM-RC-NameShort} Passport Auth
```

Comment:
Avoid at all means - module names should not be used as heathers. 
Call it passport integration and then we will need to present set of the modules that provide integration etc.